src/Turing.jl

```diff
@@ -11,6 +11,12 @@ import AdvancedVI
 using DynamicPPL: DynamicPPL, LogDensityFunction
 import DynamicPPL: getspace, NoDist, NamedDist
 import LogDensityProblems
+import NamedArrays
```

Comment:
I understand that we are importing these here so we can then grab them in the extension. 
Are they used somewhere else? If so that's fine. 
Otherwise, does the extension really need them?